package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "screwdriver-api",
   "version": "0.5.0",
-  "description": "API server for the Screwdriver CD service",
+  "description": "API server for the Screwdriver.cd service",
   "main": "index.js",
   "scripts": {
     "pretest": "eslint .",
     "test": "jenkins-mocha --recursive --timeout 4000 --exit",
     "start": "./bin/server",
-    "functional": "cucumber-js --format=pretty --tags 'not @ignore'",
+    "functional": "cucumber-js --format=progress --tags 'not @ignore' --retry 2 --fail-fast",
```

Comment:
`pretty` format was removed and we cannot see the detailed test progress even if we use other formats.